recipes/leveldb/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "1.18" %}
+
+package:
+  name: leveldb
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/google/leveldb/archive/v{{ version }}.tar.gz
+  sha1: 18684a0ad7a07920d10f5295b171fbf5eeec7337
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - snappy
```

Comment:
Saw some mentions that having `snappy` was helpful. As this is in `defaults`, figured we could just grab it.